### DISCUSSION TITLE
Any plans to open the www repository?

### DISCUSSION THREAD

**Discussion by virtualroot** [riverqueue/river#448] [General]
Hello folks

Yesterday, I found a broken link in https://riverqueue.com/docs/leader-election (shutting down), but I couldn't push a pull request with a fix.

Do you have plans to open the website repository in the future? Thanks!

**Comment by brandur** [comment]
Hey Alejandro, yeah, we should open source this at some point. It's not for now because all the docs code is intertwined with the main site, and would need to have some serious clean up and refactoring before open sourcing.

For the time being, please keep doing what you're doing an opening a discussion/issue! I just fixed this particular link.
